fiaas_skipper/config.py

```diff
@@ -52,10 +52,6 @@ def _parse_args(self, args):
                             action="store_true")
         parser.add_argument("--port", help="Port to use for the web-interface (default: %(default)s)", type=int,
                             default=5000)
-        parser.add_argument("--enable-tpr-support", help="Enable Third Party Resource support.",
-                            action="store_true")
-        parser.add_argument("--enable-crd-support", help="Enable Custom Resource Definition support.",
-                            action="store_true")
```

Comment:
These flags are used in the helm chart, so they probably have to be removed from there as well, along with the associated values: https://github.com/fiaas/skipper/blob/a19f9df2fa78c2201101b2f736281c65d6130331/helm/fiaas-skipper/templates/config_map.yaml#L28-L29

Reply:
Good catch. Should be fixed now.